src/components/AudioPlayer/AudioProgress/AudioProgress.container.js

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from 'react';
+import propTypes from 'prop-types';
+
+import {
+  convertDurationToProgress,
+  convertInputToProgress,
+  convertProgressToDuration,
+} from './songProgress';
+
+import AudioProgressComponent from './AudioProgress.component';
+
+const AudioProgressContainer = ({ audioRef }) => {
+  const [progress, setProgress] = useState(0);
+  const [isBeingDragged, setIsBeingDragged] = useState(false);
+
+  const handleDragging = () => setIsBeingDragged(true);
+
+  const handleStopDragging = () => {
+    const { duration } = audioRef.current;
+    const nextProgress = convertProgressToDuration(progress, duration);
+    // eslint-disable-next-line no-param-reassign
+    audioRef.current.currentTime = Number.isNaN(nextProgress) ? 0 : nextProgress;
```

Comment:
I don't understand how the `nextProgress` can be `NaN`.

Reply:
yeah, this is a work around for another issue, how can you duplicate that issue?